physx/source/foundation/include/unix/neon/PsUnixNeonInlineAoS.h

```diff
@@ -3567,7 +3567,6 @@ PX_FORCE_INLINE VecU32V V4U32SplatElement(VecU32V a)
 template <int index>
 PX_FORCE_INLINE Vec4V V4SplatElement(Vec4V a)
 {
-#if PX_UWP
```

Comment:
This is the code problem that I had to fix by hand to get things to build. We never figured out what changed that caused vdupq_lane_f32 to fail. I used to get builds from this source. But the solution looks good to me. 👍

Reply:
I'm not sure the [0,1] limit is an ARM specific for vdupq_lane_f32 but the code paths were essentially the same(assuming index is never below zero) so I figured it should be okay to just remove the complexity.

Reply:
vdupq_lane_f32 is ARM NEON only - and it is defined as needing to take 0 or 1 for that argument - so this should be ok. I agree that the complexity isn't necessary - I'm not sure why Physx kept it. Possible there is a minor speed bump by doing it this way - but I would guess not.

Reply:
this bit of code could actually just be a switch statement right?